src/com/oltpbenchmark/util/PlacementPolicy.java

```diff
@@ -0,0 +1,41 @@
+package com.oltpbenchmark.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/*
+ * For GeoPartitioned clusters, the configuration specifying how the
+ * tables should be placed is a list of cloud.region.zone.min_number_of_replicas
+ * blocks in addition to the total replication factor.
+ * 
+ * This class encapsulates such a placement policy.
+ */
+ 
+public class PlacementPolicy {
+    int num_replicas;
```

Comment:
use camelCase for multi words naming

Reply:
This class is used by Gson to convert to JSON and use to created tablespace syntax. So I need this field name to have the exact same name used by the SQL syntax. I have added this as a header comment in this file.